awscli/examples/schemas/create-discoverer.rst

```diff
@@ -0,0 +1,20 @@
+**To create a discoverer**
+
+The following ``create-discoverer`` creates a discoverer. ::
+
+	aws schemas create-discoverer \
```

Comment:
All your examples are using tab indenting. Indents should be exactly 4 spaces per level of indentation. Please update all your examples to use 4 space indenting.
https://aws.github.io/aws-cli/docs_styleguide.html#general

Reply:
updated spaces